### DISCUSSION TITLE
Rotary encoders with LED ring

### DISCUSSION THREAD

**Discussion by antz29** [shanteacontrols/OpenDeck#89] [Q&A]
Really sorry if this has been answered already, I do promised I was searching all over before I posted this! 

Is it possible to do something like [this](https://proto-pic.co.uk/product/rotary-encoder-led-ring-breakout-board-coloured/?attribute_pa_colour=blue&gclid=Cj0KCQjw24qHBhCnARIsAPbdtlIUYjqaRocWHvhbUqZQGjS9fxI02VgLqJZVBo7hCWZrB4od4UMQ2gMaAhlmEALw_wcB)?

I'm looking for a way to use the LED indicators to display the value of the encoder. 

I'm not fixed on using this specific control, but just more the way it's laid out. It seems that I would very quickly run out of LED outputs with event a very small number of these? Unless I'm completely missing something! (not unlikely).

I ordered my OpenDeck today so really can't wait to bring my controller to life!

John

**Comment by paradajz** [comment]
Hi,

OpenDeck board supports 48 LEDs. So if you want to use something like that you can use only 3 of them. Note that the module you've linked is not supported. LEDs on OpenDeck board should be plugged in directly.

Besides, I have never used encoders with LED rings so I don't understand how that setup works in terms of what MIDI message triggers what LED.

**Reply by antz29** [reply]
Thanks for the reply, have you got anyone using anything like VU meters etc?

John

**Reply by paradajz** [reply]
Long time ago I've used that myself. https://www.youtube.com/watch?v=ObLED808kKk
I've mapped the deck volume output to several LEDs - each LED represented percentage of the volume (that is, range 0-127). Don't remember the details anymore but it was just a mapping thing, nothing special on the config side other than assigning different control value to each LED.

**Comment by nicolasvair** [comment]
Hi, resucsitating a thread ! 

I'm about to buy a M board. 

Amont other things, i'd like to do the same thing with this led ring
https://www.duppa.net/shop/rgb-led-ring-small/

I was wondering if it would be possible to implement this as it is I2C
One I2C bus could control 16 led rings, it would be awesome for a visual feedback of the rotaries encoders.

Maybe a custom firmware ? I'd be willing to send you some led rings to try !
